internal/integration/unified/testrunner_operation.go

```diff
@@ -186,6 +186,15 @@ func executeTestRunnerOperation(ctx context.Context, operation *operation, loopD
 				}
 			}
 		}
+		return nil
+	case "wait":
```

Comment:
This is scheduled to be added in GODRIVER-2466 , but this part is required for the commaned-execution.json UST.